### PR TITLE
[proftpd] Initial integration

### PR DESCRIPTION
**Pull request by AdamKorcz**
The PR provides initial integration of [proftpd](https://github.com/proftpd/proftpd).

Examples of previous CVE's can be found here: https://github.com/proftpd/proftpd/search?q=cve&type=issues

### CODE REVIEW

File: projects/proftpd/project.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/proftpd/proftpd"
+main_repo: "https://github.com/proftpd/proftpd"
+language: c
+primary_contact: ""
+auto_ccs:
+  - "Adam@adalogics.com"
+sanitizers:
+  - address
+  - undefined
+  - memory
```
Comment (Castaglia) [inline projects/proftpd/project.yaml:10]: Given the number of uninstrumented external libraries that ProFTPD uses, I would not recommend including the memory sanitizer, at least not initially.

File: projects/proftpd/project.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/proftpd/proftpd"
+main_repo: "https://github.com/proftpd/proftpd"
+language: c
+primary_contact: ""
```
Comment (Castaglia) [inline projects/proftpd/project.yaml:4]: For the primary contact, you can use my gmail address: castaglian@gmail.com.

**Comment by AdamKorcz** [comment]
@oliverchang @jonathanmetzman @inferno-chromium Ready for review.

**Review by asraa — APPROVED** [review]
Thanks! Hope to see more tests that cover some historic CVE areas!

**Pull request merged**